packages/babel-parser/test/fixtures/estree/typescript/import-require/output.json

```diff
@@ -43,6 +43,7 @@
             "column": 32
           }
         },
+        "importKind": "value",
```

Comment:
I don't think we want to set `importKind` on `TSImportEqualsDeclaration`s

Reply:
For my own understanding, why is that? Isn't it an imported value?